projects/knora-ui/src/lib/viewer/operations/display-edit/display-edit.component.html

```diff
@@ -16,6 +17,8 @@
     </kui-interval-value>
     <kui-time-value #displayVal *ngSwitchCase="constants.TimeValue" [mode]="mode" [displayValue]="displayValue">
     </kui-time-value>
+    <kui-link-value #displayVal *ngSwitchCase="constants.LinkValue" [mode]="mode" [displayValue]="displayValue"
```

Comment:
My idea was to have an additional input with the type restriction of the linked resource. I think you can get that type from the definition, there is no need for the current value to be involved. Then it also works for create,

Reply:
you just need to know about the property, you don't need an existing value.

Reply:
@tobiasschweizer to restrict the linked resource to the ObjectType of the link property, I need to know the properties of the parent resource. that is why the parent resource should be passed as the parameter.

Reply:
see https://github.com/dasch-swiss/knora-ui-ng-lib/pull/27#discussion_r393775872